editor/register_editor_types.cpp

```diff
@@ -174,6 +175,7 @@ void register_editor_types() {
 	GDREGISTER_CLASS(EditorResourcePicker);
 	GDREGISTER_CLASS(EditorScriptPicker);
 	GDREGISTER_ABSTRACT_CLASS(EditorUndoRedoManager);
+	GDREGISTER_CLASS(PropertySelector);
```

Comment:
Good idea! Would this involve doing a refactor throughout the whole codebase? I can do that, it will just add a lot more files changed to the PR.

Reply:
It may be better to wait for the Editor team to offer their opinion since this is not my area. But I think if we decide to rename the class it should likely be done in this PR.